tests/test_images.sh

```diff
@@ -486,6 +488,12 @@ show_rootfs_metadata() {
 #
 test_distros()
 {
+	# Do some debug here
+	typeset -a temp
+	temp=()
+	for i in ${temp[@]:-}; do
+		info "defined array!"
```

Comment:
Was this actual debug, or still a valid part of the test?

Reply:
Sorry, This is debug. I'll get back to ci some time later